src/DynamoCoreWpf/Views/GuidedTour/WebBrowserWindow.xaml.cs

```diff
@@ -61,10 +63,16 @@ private void LoadWebBrowser(HtmlPage htmlPage)
 
             bodyHtmlPage = LoadResouces(bodyHtmlPage, htmlPage.Resources);
             bodyHtmlPage = LoadResourceAndReplaceByKey(bodyHtmlPage, "#fontStyle", mainFontStylePath);
+            bodyHtmlPage = SetEnconding(bodyHtmlPage);
```

Comment:
@RobertGlobant20 @filipeotero Where do the bodyHtmlPages come from? Can we change encoding from the source?

Reply:
They come from HTML files in the project. 
The reason why I'm setting this programmatically it's because there is an HTML file for every localization, so I thought this way would be easier.

Reply:
hmm @filipeotero I thought if the main html is updated, all localized html will also be updated

Reply:
@filipeotero let me know if you want to discuss this

Reply:
So I will just add the tag in the HTMLs, then the rest will be updated, right?

Reply:
@filipeotero Yes.

Reply:
@filipeotero Can we save the code here and open another PR, just in case localization team could not make it?

Reply:
Sure!

Reply:
Opened this PR for that: #12464